lib/chromedriver.js

```diff
@@ -32,7 +33,7 @@ class Chromedriver extends events.EventEmitter {
   async initChromedriverPath () {
     if (this.executableVerified) return;
     let binPath = this.chromedriver || (await getChromedriverBinaryPath());
-    if (!(await exists(binPath))) {
+    if (!(await fs.exists(binPath))) {
```

Comment:
didn't we decide that we would do `(!await fs.exists(binPath))` since precedence allows us to?